src/Popover/Popover.js

```diff
@@ -130,14 +130,16 @@ class Popover extends Component {
     this.handleResize = throttle(this.setPlacement, 100);
     this.handleScroll = throttle(this.setPlacement.bind(this, true), 50);
 
+    this.popoverRefs = {};
+
     this.state = {
       open: props.open,
       closing: false,
     };
   }
 
   componentDidMount() {
-    this.setPlacement();
+    setTimeout(this.setPlacement, 1);
```

Comment:
Yep not ideal code, but yea I'm just hoping to get the pre-v1 branch compatible before V1 catches up with some of the components. Even if it's slightly dirty.

Okay cool, I'll make those changes and do the clear on unmount. Just figured it would be over so quick that clearing it is irrelevant - but I guess these things all work on a much more micro scale.